common/util.go

```diff
@@ -530,3 +531,11 @@ func ParseEthAddr(strJsonKey string) (string, error) {
 	}
 	return "", errors.New("Error parsing address from keyfile")
 }
+
+func ValidateServiceURI(serviceURI *url.URL) bool {
+	if strings.Contains(serviceURI.Host, "0.0.0.0") {
```

Comment:
It is a valid in the context that the Orchestrator can bind to it and receive it's own ping request. However, it appears that GetOrchestrator requests will cache this IP and it can result in streams not being processed correctly when an orchestrator is selected. 

I appreciate the feedback, perhaps we can look closer at the GetOrchestrator requests to return a public IP instead of the IP defined here. This change only adds a warning to startup and prevents setting the `-serviceAddr` to `0.0.0.0` in livepeer_cli so it shouldn't cause any breaking changes, and I do not see a case where an orchestrator would use this value onchain due to the current behavior of the gateway's routing

Reply:
Ok, I think you're right